plugins/PrivacyManager/Controller.php

```diff
@@ -187,8 +188,14 @@ public function privacySettings()
         if (Piwik::hasUserSuperUserAccess()) {
             $jsCodeGenerator = new TrackerCodeGenerator();
             $file = new File(PIWIK_DOCUMENT_ROOT . '/' . $jsCodeGenerator->getJsTrackerEndpoint());
+            $filename = $jsCodeGenerator->getJsTrackerEndpoint();
 
-            $view->trackerFileName = $jsCodeGenerator->getJsTrackerEndpoint();
+            if (Manager::getInstance()->isPluginActivated('CustomJsTracker')) {
+                $file = StaticContainer::get('Piwik\Plugins\CustomJsTracker\TrackerUpdater')->getToFile();
```

Comment:
just seeing we should probably in https://github.com/matomo-org/matomo/blob/4.1.0-b1/plugins/CustomJsTracker/TrackerUpdater.php#L52  call `$jsCodeGenerator->getJsTrackerEndpoint()` as well to get the correct endpoint for historical instances. It would now check for `matomo.js` being writable when many maybe use still `piwik.js`. `TrackerUpdater` itself wouldn't need this change as it tries to update both files anyway. But might be useful here (maybe also in the system check where we check if the tracker file is writable). 

But then there are likely also systems by now where Matomo would check for `piwik.js` but they now use `matomo.js` 😕  I suppose we can also just leave it as it is and assume that if one of them is writable that both are. It might be making more sense to check for `matomo.js`. Old systems who have `piwik.js` writable but not `matomo.js` would then get notified basically that the file will need write permission. 

So all good we don't need to change anything just wanted to put my thinking in writing to figure things out :)